runtime/src/bank.rs

```diff
@@ -8441,13 +8630,34 @@ pub(crate) mod tests {
         assert!(bank.stakes.read().unwrap().vote_accounts().is_empty());
         assert!(bank.stakes.read().unwrap().stake_delegations().is_empty());
         assert_eq!(bank.calculate_capitalization(), bank.capitalization());
+        assert_eq!(
```

Comment:
these additional assertions to a test added at #11750 exposes a regression introduced at #13403.